tasks/install.yml

```diff
@@ -30,12 +40,16 @@
   when:
     downloaded_pkg.status_code is defined and downloaded_pkg.status_code == 200
 
-- name: Confluence | Checking confluence home path
+- name: Confluence | Checking confluence paths permissions
   file:
-    path: "{{ confluence_home_path }}"
+    path: "{{ item }}"
     state: directory
     owner: "{{ confluence_user }}"
     group: "{{ confluence_group }}"
+    recurse: yes
+  with_items:
```

Comment:
(I'm ok with this, but just as a comment) why not `loop` instead of `with_items`?

Reply:
just habits, both of them would work